namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-court-data-adaptor-test/resources/messaging-queues.tf

```diff
@@ -1,3 +1,66 @@
+resource "random_id" "id" {
+  byte_length = 6
+}
+
+data "aws_iam_policy_document" "laa_crime_apps" {
+  statement {
+    sid     = "PublishPolicy"
+    effect  = "Allow"
+    actions = ["sqs:SendMessage"]
+
+    resources = [
+      "${module.create_link_queue.sqs_arn}",
+      "${module.unlink_queue.sqs_arn}",
+      "${module.laa_status_update_queue.sqs_arn}",
+      "${module.hearing_resulted_queue.sqs_arn}",
+      "${module.prosecution_concluded_queue.sqs_arn}",
+    ]
+
+    principals {
+      type        = "AWS"
+      identifiers = ["*"]
+    }
+  }
+
+  statement {
+    sid     = "ConsumePolicy"
+    effect  = "Allow"
+    actions = ["sqs:ReceiveMessage"]
+
+    resources = [
+      "${module.create_link_queue.sqs_arn}",
+      "${module.unlink_queue.sqs_arn}",
+      "${module.laa_status_update_queue.sqs_arn}",
+      "${module.hearing_resulted_queue.sqs_arn}",
+      "${module.prosecution_concluded_queue.sqs_arn}",
+    ]
+
+    principals {
+      type        = "AWS"
+      identifiers = ["842522700642"]
```

Comment:
The Identifier is different from what the removed sqs policy has `013163512034`

Reply:
yeah, I only just realised AWS account IDs are all mixed up.

Reply:
Is this a better approach? https://github.com/ministryofjustice/cloud-platform-environments/pull/5541/files#diff-eddf3da7a87693d0f9ed1def33d3de9a1f6aebca07e6b30c6a68e693ba8e069eR42